pkg/model/local_target.go

```diff
@@ -0,0 +1,66 @@
+package model
+
+import (
+	"fmt"
+
+	"github.com/windmilleng/tilt/internal/sliceutils"
+)
+
+type LocalTarget struct {
+	Cmd  Cmd
+	Deps []string
+
+	dockerignores []Dockerignore
+	repos         []LocalGitRepo
+}
+
+var _ TargetSpec = LocalTarget{}
+
+func NewLocalTarget(cmd Cmd, deps []string) LocalTarget {
```

Comment:
it would be good to clarify that `deps` have to be absolute directories (since they're fulfilling the Dependencies() contract)